src/main/webapp/content/scss/partials/_filter-pane.scss

```diff
@@ -30,6 +30,12 @@
   .favouritefilter {
     margin: auto;
   }
+  i.favouritefilter.fa.fa-star{
+      color: $blue;
```

Comment:
remove blue var color with primary_color..
pls note that $primary_color always use this color as it is primary color of app

Reply:
done